tests/lib/rules/no-unused-expressions.js

```diff
@@ -82,6 +82,20 @@ ruleTester.run("no-unused-expressions", rule, {
         {
             code: "obj?.foo(\"bar\")",
             parserOptions: { ecmaVersion: 11 }
+        },
+
+        // JSX
+        {
+            code: "<div />",
+            parserOptions: { ecmaFeatures: { jsx: true } }
+        },
+        {
+            code: "<></>",
+            parserOptions: { ecmaFeatures: { jsx: true } }
+        },
+        {
+            code: "var partial = <div />",
+            parserOptions: { ecmaFeatures: { jsx: true } }
```

Comment:
Can you add a pair of valid tests for an element and a fragment that opt in to the option?